resources/views/backend/hotels/edit.blade.php

```diff
@@ -0,0 +1,134 @@
+@extends('backend.layouts.main')
+@section('title', __('Create Hotel'))
+@section('content')
+  <div class="content-wrapper">
+    <section class="content">
+    <h1 class="title-page text-success">{{__('Update hotel')}}</h1>
+      <div class="row margin-center">
+        <div class="col-md-12">
+          <div class="box box-primary">
+            <div class="box-header with-border">
+              <h3 class="box-title lead">{{ __('Enter information') }}</h3>
+            </div>
+            <form role="form" method="POST" action="{{ route('hotel.update', $hotel->id) }}" enctype="multipart/form-data" >
+              {{csrf_field()}}
+              {{method_field('PUT')}}
+              <input type="hidden" value="{{ $hotel->id }}" name="id">
+              <div class="box-body"> 
+                {{-- input hotel name --}}
+                <div class="form-group" {{ $errors->has('name') ? ' has-error' : '' }}>
+                  <label>{{ __('Name') }}</label>
+                  <input type="text" class="form-control" name="name" value="{{ $hotel->name }}">
+                  @if($errors->first('name'))
+                    <span class="text-danger">{{ $errors->first('name') }}</span>
+                  @endif
+                </div>
+                {{-- input address --}}
+                <div class="form-group" {{ $errors->has('address') ? ' has-error' : '' }}>
+                  <label>{{ __('Address') }}</label>
+                  <input type="text" class="form-control" name="address" value="{{ $hotel->address }}">
+                  @if($errors->first('address'))
+                    <span class="text-danger">{{ $errors->first('address') }}</span>
+                  @endif
+                </div>
+                {{-- select place adn star --}}
+                <div class="form-inline">
+                  {{-- place select --}}
+                  <div class="form-group" {{ $errors->has('place') ? ' has-error' : '' }}>
+                    <p><strong>{{ __('Place') }}</strong></p>
+                    <select class="form-control" name="place_id">
+                      <option value="">{{ __('Choose Place') }}</option>
+                      @foreach($places as $place)
+                        <option value="{{ $place->id }}" {{$hotel->place_id == $place->id ? 'selected' :''}}>{{ $place->name }}</option>
```

Comment:
Two thing here:
- One space after `:`.
- Should unify only one kind when put varriable in `{{ ... }}`.
  1. Wrapping by space: `{{ $varA }}`
  2. No wrapping by space: `{{$varB}}`
  I think we should use `#1` standard.